src/data/accessibility-quiz.ts

```diff
@@ -1,4 +1,22 @@
 const accessibilityQuiz = [
+  {
+    "Question": "What is semantic HTML?",
+   "Answer": "HTML that uses meaningful tags to structure content",
+   "Distractor1": "HTML that uses only <div> and <span> tags",
+   "Distractor2": "HTML that is styled with CSS",
+   "Distractor3": "HTML that includes JavaScript",
+   "Explanation": "Semantic HTML uses meaningful tags such as <header>, <footer>, and <article> to improve the readability and accessibility of web content.",
+   "Link": "https://www.freecodecamp.org/news/semantic-html5-elements/"
+  },
```

Comment:
We should remove this question because it is really similar to the second question you wrote. And the second question is stronger. So we will just go with that one.

Reply:
Thank you so much. Alright so do I have to remove the codes on line 2 to line 10 I made earlier or can you do it from your end?

Reply:
Okay, I have updated the file by doing what you suggested.